packages/frontend/src/components/message/MessageMetaData.tsx

```diff
@@ -54,6 +56,12 @@ export default function MessageMetaData(props: Props) {
           className={'padlock-icon'}
         />
       )}
+      {isSavedMessage && (
+        <div
+          aria-label={tx('saved_message')}
```

Comment:
better use the existing string here. the context of "message" is anyway clear

```suggestion
          aria-label={tx('saved')}
```